volume-providers/vpc/vpcclient/vpcvolume/list_volume_tags.go

```diff
@@ -14,11 +14,15 @@ import (
 	"github.com/IBM/ibmcloud-storage-volume-lib/lib/utils"
 	"github.com/IBM/ibmcloud-storage-volume-lib/volume-providers/vpc/vpcclient/client"
 	"github.com/IBM/ibmcloud-storage-volume-lib/volume-providers/vpc/vpcclient/models"
+	"go.uber.org/zap"
 	"time"
 )
 
 // ListVolumeTags GETs /volumes/tags
-func (vs *VolumeService) ListVolumeTags(volumeID string) (*[]string, error) {
+func (vs *VolumeService) ListVolumeTags(volumeID string, ctxLogger *zap.Logger) (*[]string, error) {
+	ctxLogger.Info("Entry Backend ListVolumeTags")
```

Comment:
Can we have `Debug` type instead of `Info`

Reply:
DONE